packages/styles/scss/patterns/card-section/_card-section.scss

```diff
@@ -7,6 +7,8 @@
 @import '../../components/link-with-icon/link-with-icon';
 
 @mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
   .bx--card-link {
     background-color: $ui-01;
     outline: 1px solid $ui-background;
```

Comment:
I still don't think we should be using outline to do this. The condensed grid was designed to allow us to layout our cards in this way.

We're getting sone weird outlining issues.

<img src="https://user-images.githubusercontent.com/3793636/71921875-5cf5a000-314f-11ea-9d09-b8071b30c6a2.png" width="300px" />

Reply:
oh I will remove that outline.